website/source/api/agent/service.html.md

```diff
@@ -58,6 +58,66 @@ $ curl \
 }
 ```
 
+## Get local service health
+
+The `/v1/agent/health/service/name/<service_name>` and
+`/v1/agent/health/service/id/<service_id>` endpoints allow to retrieve an
+aggregated state of service(s) of the local agent.
+
+The endpoint `/v1/agent/health/service/name/<service_name>` query all
+services with a given names (several may match), while
+`/v1/agent/health/service/id/<service_id>` will match a single service only.
+
+If you know the ID of service you want to target, it is recommended to use
+the version `/v1/agent/health/service/id/<service_id>` so you have the result
+for the service only. When requesting
+`/v1/agent/health/service/name/<service_name>`, the caller will receive the
+worst state of all services having the given name.
+
+Those endpoints return the aggregated values of all healthchecks for the
+service and will return the corresponding HTTP codes:
+
+| Result | Meaning                                                |
+| ------ | ------------------------------------------------------ |
+| `200`  | All healthchecks of this service are passing           |
+| `400`  | Bad parameter (missing service name of id)             |
+| `404`  | No such service id or name                             |
+| `429`  | Some healthchecks are passing, at least one is warning |
+| `503`  | All healthchecks of this service are passing           |
```

Comment:
I think there is a typo here. Shouldn't this say no healthcheck is passing or something to that effect?

Reply:
@mkeeler Well spotted, fixed